app/Services/Users/UserCreationService.php

```diff
@@ -75,6 +75,38 @@ public function handle(array $data)
             $token = $this->passwordBroker->createToken($user);
         }
 
+        if ($this->settings->get('jexactyl::approvals:webhook') === 'true' && strlen($this->settings->get('jexactyl::approvals:webhook_url')) > 0) {
```

Comment:
No need for this strlen expression. Just ensure that `webhook_url` returns a value.